doc/lsd.md

```diff
@@ -140,6 +140,9 @@ lsd is a ls command with a lot of pretty colours and some other stuff to enrich
 `--header`
 : Display block headers
 
+`--literal`
+: Do not show quotes on filenames
```

Comment:
> print entry names without quoting

ref: https://man7.org/linux/man-pages/man1/ls.1.html

Reply:
Fixed it as well according to the manual